generated/nidaqmx/task/_timing.py

```diff
@@ -12,11 +12,15 @@ class Timing:
     """
     Represents the timing configurations for a DAQmx task.
     """
-    __slots__ = ('_handle', '_interpreter')
+    __slots__ = ('_handle', '_interpreter', '_active_devs')
 
-    def __init__(self, task_handle, interpreter):
+    def __init__(self, task_handle, interpreter, active_devs=None):
         self._handle = task_handle
         self._interpreter = interpreter
+        self._active_devs = active_devs
+
+    def __getitem__(self, dev):
```

Comment:
getitem is also used for indexing, so make sure that `task.timing[3]` either does something useful or raises a reasonable exception (and not a https://en.wikipedia.org/wiki/Garbage_in,_garbage_out exception like `AttributeError: 'NoneType' object has no attribute 'name'`)